ams/server/src/main/java/com/netease/arctic/server/manager/MetricsManager.java

```diff
@@ -61,4 +67,26 @@ public void emit(MetricsContent<?> metrics) {
       }
     });
   }
+
+  static class MetricsContentWrapper {
```

Comment:
I think for Iceberg or Paimon metric wrapper , it is reasonable to declare single classes for them